src/open_prime_rando/echoes/widescreen_hud/__init__.py

```diff
@@ -0,0 +1,30 @@
+from retro_data_structures.base_resource import RawResource
+
+from open_prime_rando.echoes.custom_assets import custom_asset_path
+from open_prime_rando.echoes.widescreen_hud.asset_map import WIDESCREEN_HUD_ASSETS
+from open_prime_rando.patcher_editor import PatcherEditor
+
+
+def apply_widescreen_hud(editor: PatcherEditor):
+    """
+    Replaces certain FRMEf files to adjust HUD/Visor widgets to better fit a 16:9 screen aspect ratio
+    """
+    if editor.does_asset_exists(0xEEF43AA1) and editor.does_asset_exists(0xF7EC0850):
+        detectedversion = "ntscu"
+    elif editor.does_asset_exists(0xB5CF0C19) and editor.does_asset_exists(0xD9D58FA5):
+        detectedversion = "pal"
+    else:
+        raise NotImplementedError
```

Comment:
again, don't love this. we already have some amount of version handling for dol patches and I'd like to see that leveraged unless there's a very specific reason it needs to be handled this way (in which case please add a comment explaining that reason)

Reply:
https://github.com/randovania/open-prime-rando/pull/108/commits/a7be495d7c3b8e86c227f7d2672c79e740d4832d

Add comment explaining manual version detection 
```py
 # TODO: Properly do version detection
 # Currently detecting game version by existence of version-specific assets here, since PAK patching doesn't seem
 # to have version detection,unlike DOL patching that does
 ```

Reply:
https://github.com/randovania/open-prime-rando/pull/108/commits/6dfe9451f1d4c24519573b872698989fc88ce5f7
Changed to no longer need the notion of versioning - if the asset exists in game, replace it. If it doesn't ignore it.

I checked to make sure that the custom asset IDs are different if the files are different, so there's no problematic overlap between the versions. ([My Notes](https://github.com/Nystrata/EchoesWidescreenHUD/wiki/Notes#gguisyspak))